ukis_pysat/data.py

```diff
@@ -85,54 +99,23 @@ def query_metadata(self, platform, date, aoi, cloud_cover=None):
             (yyyyMMdd, yyyy-MM-ddThh:mm:ssZ, NOW, NOW-<n>DAY(S), HOUR(S), MONTH(S), etc.)
         :param aoi: Area of interest as GeoJson file or bounding box tuple with lat lon coordinates (String, Tuple).
         :param cloud_cover: Percent cloud cover scene from - to (Integer tuple).
-        :returns: Metadata of products that match query criteria (List of Metadata objects).
+        :returns: Metadata catalog of products that match query criteria (PySTAC Catalog).
         """
         if self.src == Datahub.File:
-            # query Filesystem for metadata

```

Comment:
This is essentially the part where we query a directory for valid metadata items. We should move this to STAC.

Reply:
This would be part of the fill_catalog_from_directory() method.